content/22.vaccines.md

```diff
@@ -176,7 +179,7 @@ As of December 11, 2020, the United States FDA approved this vaccine under an em
 
 ### Viral Vector Vaccines
 
-#### COVID-19 Vaccine AstraZeneca
+<!---- #### COVID-19 Vaccine AstraZeneca --->
```

Comment:
Is this the reorganization you referred to?

Reply:
Yes also creating a new subsection header `#### INO-4800 Vaccine` so that the DNA section followed the same format of background then example vaccines.

I commented out some of the examples since we are a bit pressed for time and so we wanted to at least have 1 vaccine example per section

Reply:
Got it, thanks!